python/cugraph/cugraph/traversal/bfs.py

```diff
@@ -42,8 +41,9 @@ def _ensure_args(G, start, i_start, directed):
     start = start if start is not None else i_start
 
     G_type = type(G)
+
     # Check for Graph-type inputs
-    if G_type is Graph or is_nx_graph_type(G_type):
+    if G_type is Graph:
```

Comment:
Probably also more correct to use `isinstance()` and remove `G_type = type(G)`, here and in all other places.